src/site/content/en/blog/optimize-fid/index.md

```diff
@@ -0,0 +1,300 @@
+possible, use
+[Brotli](https://opensource.googleblog.com/2015/09/introducing-brotli-new-compression.html) which
+provides better compression results than
+[Gzip](https://www.youtube.com/watch?v=whGwm0Lky2s&feature=youtu.be&t=14m11s) and [can be used in
+almost all newer browsers](https://caniuse.com/#feat=brotli).
+
+{% Aside %}
+For more details, refer to the [Minify and compress network payloads](/reduce-network-payloads-using-text-compression/) guide.
+{% endAside %}
+
+### Defer unused JavaScript
+
+By default all JavaScript is render-blocking. When the browser encounters a script tag that links to
+an external JavaScript file, it must pause what it's doing and download, parse, compile, and execute
+that JavaScript. Therefore you should only load the code that's needed for loading the page or
+responding to user input.
+
+The [Coverage](https://developers.google.com/web/tools/chrome-devtools/coverage) tab in Chrome
+DevTools can tell you how much JavaScript is not being used on your web page.
+
+![Coverage tab in Chrome DevTools](./coverage-panel-js.png)	
+
+To cut down on unused JavaScript:
+
++   Code-split your bundle into multiple chunks
++   Defer any non-critical JavaScript, including third-party scripts, using `async` or `defer`
+
+**Code-splitting** is the concept of splitting a single large JavaScript bundle into smaller chunks
+that can be conditionally loaded (also known as lazy-loading).
+[Most newer browsers support dynamic import syntax](https://caniuse.com/#feat=es6-module-dynamic-import),
+which allows for module fetching on demand:
+
+```js	
+import('module.js')	
+  .then((module) => {	
+    // Do something with the module.	
+  });	
+```
+
+Dynamically importing JavaScript on certain user interactions (such as changing a route or
+displaying a modal) will make sure that code not used for the initial page load is only fetched when
+needed.
+
+Aside from general browser support, dynamic import syntax can be used in many different build
+systems.
+
++   If you use [webpack](https://webpack.js.org/guides/code-splitting/),	
+    [Rollup](https://medium.com/rollup/rollup-now-has-code-splitting-and-we-need-your-help-46defd901c82),	
+    or [Parcel](https://parceljs.org/code_splitting.html) as a module bundler, take advantage of	
+    their dynamic import support. 	
++   Client-side frameworks, like	
+    [React](https://reactjs.org/docs/code-splitting.html#reactlazy),	
+    [Angular](https://angular.io/guide/lazy-loading-ngmodules), and	
+    [Vue](https://vuejs.org/v2/guide/components-dynamic-async.html#Async-Components) provide	
+    abstractions to make it easier to lazy-load at the component-level.
+
+{% Aside %}
+Take a look at [Reduce JavaScript payloads with code
+splitting](/reduce-javascript-payloads-with-code-splitting/) to learn more about code-splitting.
+{% endAside %}
+
+Aside from code-splitting, always use [async or	
```

Comment:
Hmmm, I mean this is good advice, but I'm not sure `async` and `defer` it'll reduce FID.